src/Faithlife.Data/DbValueTypeInfo.cs

```diff
@@ -30,9 +30,25 @@ internal static DbValueTypeStrategy GetStrategy(Type type)
 			if (type.GetTypeInfo().IsEnum)
 				return DbValueTypeStrategy.Enum;
 
+			if (IsRecordLike(type))
```

Comment:
This wouldn't be a meaningful change in behavior because it will only use the positional record strategy for a class with exactly one constructor and where that constructor has parameters that match the class's properties. The behavior prior to the change requires a zero-parameter constructor.

Reply:
Good point!